tests/Ylmish.Tests/common/Example.g.fs

```diff
@@ -0,0 +1,67 @@
+            (unbox<AdaptiveSubmodel> o).Update(v)
+            o
+        Adaptify.FSharp.Core.AdaptiveOption<Example.Submodel, Example.AdaptiveSubmodel, Example.AdaptiveSubmodel>(value.PropF, (fun (v : Submodel) -> AdaptiveSubmodel(v) :> System.Object), __arg2, (fun (o : System.Object) -> unbox<AdaptiveSubmodel> o), (fun (v : Submodel) -> AdaptiveSubmodel(v) :> System.Object), __arg5, (fun (o : System.Object) -> unbox<AdaptiveSubmodel> o))
+    let mutable __value = value
+    let __adaptive = FSharp.Data.Adaptive.AVal.custom((fun (token : FSharp.Data.Adaptive.AdaptiveToken) -> __value))
+    static member Create(value : Model) = AdaptiveModel(value)
+    static member Unpersist = Adaptify.Unpersist.create (fun (value : Model) -> AdaptiveModel(value)) (fun (adaptive : AdaptiveModel) (value : Model) -> adaptive.Update(value))
+    member __.Update(value : Model) =
+        if Microsoft.FSharp.Core.Operators.not((FSharp.Data.Adaptive.ShallowEqualityComparer<Model>.ShallowEquals(value, __value))) then
+            __value <- value
+            __adaptive.MarkOutdated()
+            _PropA_.Value <- value.PropA
+            _PropB_.Value <- value.PropB
+            _PropC_.Update(value.PropC)
+            _PropD_.Value <- value.PropD
+            _PropE_.Update(value.PropE)
+            _PropF_.Update(value.PropF)
+    member __.Current = __adaptive
+    member __.PropA = _PropA_ :> FSharp.Data.Adaptive.aval<Microsoft.FSharp.Core.string>
+    member __.PropB = _PropB_ :> FSharp.Data.Adaptive.aval<Microsoft.FSharp.Core.option<Microsoft.FSharp.Core.string>>
+    member __.PropC = _PropC_ :> FSharp.Data.Adaptive.alist<AdaptiveSubmodel>
+    member __.PropD = _PropD_ :> FSharp.Data.Adaptive.alist<Microsoft.FSharp.Core.string>
+    member __.PropE = _PropE_
+    member __.PropF = _PropF_ :> FSharp.Data.Adaptive.aval<Adaptify.FSharp.Core.AdaptiveOptionCase<Submodel, AdaptiveSubmodel, AdaptiveSubmodel>>
+[<System.Diagnostics.CodeAnalysis.SuppressMessage("NameConventions", "*")>]
+type AdaptiveSubmodel(value : Submodel) =
+    let _Prop0_ = FSharp.Data.Adaptive.cval(value.Prop0)
+    let mutable __value = value
+    let __adaptive = FSharp.Data.Adaptive.AVal.custom((fun (token : FSharp.Data.Adaptive.AdaptiveToken) -> __value))
+    static member Create(value : Submodel) = AdaptiveSubmodel(value)
+    static member Unpersist = Adaptify.Unpersist.create (fun (value : Submodel) -> AdaptiveSubmodel(value)) (fun (adaptive : AdaptiveSubmodel) (value : Submodel) -> adaptive.Update(value))
+    member __.Update(value : Submodel) =
+        if Microsoft.FSharp.Core.Operators.not((FSharp.Data.Adaptive.ShallowEqualityComparer<Submodel>.ShallowEquals(value, __value))) then
+            __value <- value
+            __adaptive.MarkOutdated()
+            _Prop0_.Value <- value.Prop0
+    member __.Current = __adaptive
+    member __.Prop0 = _Prop0_ :> FSharp.Data.Adaptive.aval<Microsoft.FSharp.Core.string>
+
```

Comment:
Should we add these *.g.fs to .gitignore, or are we including it for browsability?

Reply:
I think it's better to keep them in git so that:

1. we can track changes as they occur, whether due to changes to the models or to Adaptify
2. it means that opening affected projects for the first time doesn't show the file as missing
 
I don't feel strongly about it though - and we may find the additional noise outweighs any advantages (though since there's only 1 file affected so far, it won't be hard to change it in future).

Reply:
Good reasons, let's keep 'em